### PR TITLE
Hook up workflow rendering to push lifecycle

### PR DESCRIPTION
**Pull request by cdupuis**

### CODE REVIEW

**Comment by cdupuis** [comment]
Pull request auto merge enabled. [atomist:enable-auto-merge]

File: test/handlers/event/push/workflow/CircleWorkflowTest.ts [hunk 1]
```diff
@@ -34,14 +34,12 @@ describe("CircleWorkflow", () => {
 
     it("construct completed workflow stages", () => {
         const workflow = {
-            teamId: "TEAM1",
             id: "workflow id",
             name: "pipelineDooling",
```
Comment (ddgenome) [inline test/handlers/event/push/workflow/CircleWorkflowTest.ts]: Hmm

**Comment by ddgenome** [comment]
I've rebased to fix the conflicts and will merge once the builds are complete.

**Comment by ddgenome** [comment]
Looks like automerge is enabled

**Pull request merged**